tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/polyparser/ParseFile.scala

```diff
@@ -85,6 +85,11 @@ object ParseFile {
     * @param parseSource the parses we want to evaluate against
     */
   def evaluateParserOnTestSet(parser: TransitionParser, parseSource: PolytreeParseSource): Unit = {
+    println("Loading lazy data structures.")
```

Comment:
I see now that this is a method for testing the parser -- so it's a little less necessary to change here :)